theano/scalar/basic_scipy.py

```diff
@@ -394,3 +394,73 @@ def __hash__(self):
         return hash(type(self))
 
 chi2sf = Chi2SF(upgrade_to_float, name='chi2sf')
+
+
+class J1(UnaryScalarOp):
+    """
+    Bessel function of the 1'th kind
+    """
+
+    @staticmethod
+    def st_impl(x):
+        return scipy.special.j1(x)
+
+    def impl(self, x):
+        if imported_scipy_special:
+            return self.st_impl(x)
+        else:
+            super(J1, self).impl(x)
```

Comment:
I mostly copy pasted from the psi class just above j0 and j1. It doesn't have a return either.